doc/additional-configurations.md

```diff
@@ -127,7 +136,7 @@ Before you get to the task of creating a new database for SonarQube, you need to
 		![](_img/New-Database-User-Mapping.png)
 	- Click **OK** to complete the new user setup.
 
-3. **Test connection**
+4. **Test connection**

```

Comment:
Testing the connection is configured correctly is useful for integrated security too.
It looks like only line 148 would have to change to some thing like "- Select the appropriate authentication method and is using Sql Authentication provide the **User Name ** and **Password** you created.